TrkReco/src/TrkRecoMcUtils_tool.cc

```diff
@@ -74,19 +74,28 @@ namespace mu2e {
 
   //-----------------------------------------------------------------------------
   TrkRecoMcUtils::TrkRecoMcUtils(const fhicl::ParameterSet& PSet) :
```

Comment:
Is this constructor still needed, and if so, why?  Can't this tool always be configured from the Config struct?

Reply:
thanks for raising this issue ! .. I tried to switch ot a config-based constructor some time ago and failed.  Spent more time now - turned out that a config-based constructor exists but takes more parameters, which is absolutely not obvious from the error diagnostics...

Anyway, the tool changed to use a config-based construction.